linera-execution/src/lib.rs

```diff
@@ -142,6 +142,8 @@ pub enum ExecutionError {
     JoinError(#[from] linera_base::task::Error),
     #[error(transparent)]
     DecompressionError(#[from] DecompressionError),
+    #[error(transparent)]
+    BcsError(#[from] bcs::Error),
```

Comment:
Is there a reason for moving this?

Reply:
I think I removed it then added it somewhere else by accident as the PR evolved. Will fix